modules/swagger-codegen/src/main/resources/python/model.mustache

```diff
@@ -52,7 +54,7 @@ class {{classname}}(object):
     }
 {{/discriminator}}
 
-    def __init__(self{{#vars}}, {{name}}={{#defaultValue}}{{{defaultValue}}}{{/defaultValue}}{{^defaultValue}}None{{/defaultValue}}{{/vars}}, _configuration=None):  # noqa: E501
+    def __init__(self{{#allVars}}, {{name}}={{#defaultValue}}{{{defaultValue}}}{{/defaultValue}}{{^defaultValue}}None{{/defaultValue}}{{/allVars}}, _configuration=None):  # noqa: E501
```

Comment:
Similarly because the model is defined with the constructor rather than using setters for each property, the parent vars must be passed in the constructor as well as the properties belonging to this model